.bazelrc

```diff
@@ -3,3 +3,5 @@ build --workspace_status_command="python tools/workplace_status.py"
 # Flags used during releases
 build:release --stamp
 test:release --stamp
+
+test --test_env=HOME
```

Comment:
Is it possible to remove this, or is it used by the tests?

Reply:
Coming into this repo, I can't run `bazel test ...` (Linux), most things fail with errors about $HOME. Happy to remove it if you'd prefer, just thought it was good for contributors.

Reply:
Huh, that's bad! Can you show me what the errors are? I don't think there is anything in the preexisting tests that depends on `$HOME`, but I've been wrong many times before

Reply:
```
$ bazel version
Bazelisk version: v1.3.0
Build label: 3.1.0
Build target: bazel-out/k8-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Wed Apr 22 10:32:27 2020 (1587551547)
Build timestamp: 1587551547
Build timestamp as int: 1587551547
$ bazel test e2e/simple/... --test_output=errors
INFO: Analyzed target //e2e/simple:go_default_test (0 packages loaded, 0 targets configured).
INFO: Found 1 test target...
FAIL: //e2e/simple:go_default_test (see /home/lewis/.cache/bazel/_bazel_lewis/3c093b90e594c7f9127ef18c3d5ae685/execroot/__main__/bazel-out/k8-fastbuild/testlogs/e2e/simple/go_default_test/test.log)
INFO: From Testing //e2e/simple:go_default_test:
==================== Test output for //e2e/simple:go_default_test:
=== RUN   TestSimpleBuild
    TestSimpleBuild: simple_test.go:93: ibazel invoked as: /home/lewis/.cache/bazel/_bazel_lewis/3c093b90e594c7f9127ef18c3d5ae685/execroot/__main__/bazel-out/k8-fastbuild/bin/e2e/simple/linux_amd64_stripped/go_default_test.runfiles/__main__/ibazel/linux_amd64_pure_stripped/ibazel --bazel_path=/usr/bin/bazel --log_to_file=/tmp/ibazel_output.307214372.log run //:simple
    TestSimpleBuild: simple_test.go:93: Unable to build target. Error code: 1
        Stdout:
        
        Stderr:
        2020/04/28 21:09:41 could not get the user's cache directory: neither $XDG_CACHE_HOME nor $HOME are defined
--- FAIL: TestSimpleBuild (0.03s)
=== RUN   TestSimpleRunAfterShutdown
    TestSimpleRunAfterShutdown: simple_test.go:105: bazel failed to shut down
--- FAIL: TestSimpleRunAfterShutdown (0.03s)
=== RUN   TestSimpleRunConfirmEnvironment
    TestSimpleRunConfirmEnvironment: simple_test.go:119: ibazel invoked as: /home/lewis/.cache/bazel/_bazel_lewis/3c093b90e594c7f9127ef18c3d5ae685/execroot/__main__/bazel-out/k8-fastbuild/bin/e2e/simple/linux_amd64_stripped/go_default_test.runfiles/__main__/ibazel/linux_amd64_pure_stripped/ibazel --bazel_path=/usr/bin/bazel --log_to_file=/tmp/ibazel_output.289262387.log run //:environment

Reply:
Ah you know what, this might be bazelisk related...

Reply:
Ok yup, pretty sure that's it, Bazelisk stores bazel installs in a cache dir under $HOME, and as the tests run bazel...

I can remove this, and add a comment to CONTRIBUTORS.md?

Reply:
Could you try updating your Bazelisk to the latest and if that doesn't fix it file a bug? I think they would be interested in it.